package.json

```diff
@@ -1,10 +1,26 @@
 {
```

Comment:
please revert the changes to `package.json` and `package-lock.json`

Reply:
I edited it to make the repo more understandable. My edit to package-lock is again only about the repo information, but I can revert it if you want. No problem. @alexjustesen